apps/mark-scan/frontend/src/pages/no_paper_handler_page.tsx

```diff
@@ -5,11 +5,8 @@ export function NoPaperHandlerPage(): JSX.Element {
     <Screen>
       <Main padded centerChild>
         <Text center>
-          <H1>No Connection to Printer-Scanner</H1>
-          <P>
-            Please restart the machine. If the problem persists, an internal
-            connection may be loose.
-          </P>
+          <H1>Internal Connection Problem</H1>
+          <P>Ask a poll worker for help with restarting the machine.</P>
```

Comment:
Should we use `appStrings` here instead of hardcoded copy?

Reply:
Yes, since this has changed from a poll worker page to a possibly voter-facing page (though I'm guessing it's much more likely this screen would show on first boot after moving the unit)